### PR TITLE
Project Setup

### PR DESCRIPTION
**Pull request by hoangmirs**
## What happened

Init project
 
## Insight

- Use rails-templates
- Upgrade to latest ruby and rails
- Enable FrozenStringLiteralComment
 
## Proof Of Work

localhost_3000_(Laptop with MDPI screen)

### CODE REVIEW

File: .semaphore/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Semaphore CI 2.0
```
Comment (olivierobert) [inline .semaphore/README.md:1]: Let's remove Semaphore set up stuff as we will be using Github Actions for this project

**Comment by olivierobert** [comment]
The branch should be `chore/init-project`, there is not such type as `task`

File: .semaphore/README.md (hunk 1 above)
Comment (hoangmirs) [reply .semaphore/README.md]: Removed in https://github.com/nimblehq/nimble-survey-web/pull/1/commits/86e0fb91c47254aab0053bcb04614daa90bd521b

**Comment by ankitkalia1195** [comment]
@hoangmirs Did you build the project in api mode from rails template, or without the api option?

**Comment by hoangmirs** [comment]
> @hoangmirs Did you build the project in api mode from rails template, or without the api option?

@ankitkalia1195 No, I generated it without API option. We have a few features that require the UI (signup, reset password,...)

**Pull request merged**